### PR TITLE
Package init

### PR DESCRIPTION
**Pull request by hubblezzhu**
Porting and rewrite BRS test from https://github.com/intel/rv-brs-test-suite

### CODE REVIEW

**Comment by xiaobo55x** [comment]
Hi Hongbo, Could you break this commit into small patch set?

Hi @avpatel, Hongbo will help on cleaning up the old BRS Test Suite repo, please help reviewing the patch!

Thanks,
Haibo

**Comment by avpatel** [comment]
@xiaobo55x Thanks!
@hubblezzhu It will be helpful if the single commit is broken down into separate patches each with its own commit description.

**Comment by hubblezzhu** [comment]
@avpatel @xiaobo55x Alright, let me break it down.

**Comment by hubblezzhu** [comment]
Hi, @xiaobo55x @avpatel  This patch had been separated into small ones. 

The first few are the construction of each submodule:
  Add buildroot build
  Add edk2 build  
  Add edk2-test build 
  Add edk2-test-parser build 
  Add grub build 
  Add linux build 
  Add opensbi build 
  Add qemu build 
  Add sbi-test build 

The last two are the scripts for image construction and UEFI testing.
  Add brs image build scripts 
  Add uefi sct test scripts

**Review by xiaobo55x — COMMENTED** [review]
Suggest to update the commit log to:

''
Enable FWTS in buildroot

The FWTS were patched with some RISC-V specific changes which were supposed to be unstreamed after ACPI 6.6 was released. 
''

**Review by xiaobo55x — CHANGES_REQUESTED** [review]
Suggest to update the commit log to:

''
Enable FWTS in buildroot

The FWTS were patched with some RISC-V specific changes which were supposed to be unstreamed after ACPI 6.6 was released.
''

**Comment by avpatel** [comment]
Lot of sources are not permissively licensed and it has some non-standard open-source license header with copyright from various companies. This needs to be reviewed from copyright or license expert.

File: src/brs-edk2-test/brsi-tests/BrsBootServices/BlackBoxTest/BrsBootServicesBBTestMain.c [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  CONSEQUENTIAL, DIRECT, INDIRECT, OR SPECIAL DAMAGES WHETHER
+  UNDER CONTRACT, TORT, WARRANTY, OR OTHERWISE, ARISING IN ANY
+  WAY OUT OF THIS OR ANY OTHER AGREEMENT RELATING TO THIS
+  DOCUMENT, WHETHER OR NOT SUCH PARTY HAD ADVANCE NOTICE OF
+  THE POSSIBILITY OF SUCH DAMAGES.
+
+  Copyright 2006 - 2016 Unified EFI, Inc. All
+  Rights Reserved, subject to all existing rights in all
+  matters included within this Test Suite, to which United
+  EFI, Inc. makes no claim of right.
```
Comment (vlsunil) [inline src/brs-edk2-test/brsi-tests/BrsBootServices/BlackBoxTest/BrsBootServicesBBTestMain.c:41]: This header (in multiple files )is pretty old. These files were originally taken and modified from UEFI SCT at which time this header existed in SCT. But not UEFI SCT has moved to better copy right text. Please use latest UEFI SCT and modify as per requirement.
REF: https://github.com/tianocore/edk2-test/commit/33a88d70f9fe7120566d17fe5d47b7affd254f07

**Comment by hubblezzhu** [comment]
> Suggest to update the commit log to:
> 
> '' Enable FWTS in buildroot
> 
> The FWTS were patched with some RISC-V specific changes which were supposed to be unstreamed after ACPI 6.6 was released. ''

Modified commit log in commit “Add buildroot build”

**Comment by hubblezzhu** [comment]
> Lot of sources don't have a permissive open-source license header. This needs to be reviewed from copyright or license expert.

Added license file and license header in src files

File: src/brs-edk2-test/brsi-tests/BrsBootServices/BlackBoxTest/BrsBootServicesBBTestMain.c (hunk 1 above)
Comment (hubblezzhu) [reply src/brs-edk2-test/brsi-tests/BrsBootServices/BlackBoxTest/BrsBootServicesBBTestMain.c]: Updated the copy right header in UEFI SCT files

**Review by vlsunil — APPROVED** [review]
LGTM

**Pull request merged**